bftengine/include/bftengine/DbCheckpointManager.hpp

```diff
@@ -214,6 +201,8 @@ class DbCheckpointManager {
   concordMetrics::GaugeHandle lastDbCheckpointSizeInMb_;
   concordMetrics::GaugeHandle lastDbCheckpointBlockId_;
   concordMetrics::CounterHandle numOfDbCheckpointsCreated_;
+  std::string page;
```

Comment:
1. Why do you need this member? (please document)
2. Please conform to the naming rule (_ suffix)

Reply:
1. a reserved page is in size of 4K, it makes sense to allocate it only once instead of allocating it every time we want to write to the reserved page
2.  fixed